src/main/java/seedu/duke/util/Parser.java

```diff
@@ -158,24 +176,20 @@ public Command parseCommand(String userInput) {
     private Tag handleTagPrefix(String[] tagMessage) throws SystemException {
         String tagName;
         String tagColor = "";
+        String tagsInfo;
 
         // Ensures that the message is not blank.
         try {
-            if (tagMessage[1].isBlank()) {
-                throw new SystemException(ExceptionType.EXCEPTION_MISSING_TAG);
-            }
+            tagsInfo = checkBlank(tagMessage[1], ExceptionType.EXCEPTION_MISSING_TAG);
         } catch (ArrayIndexOutOfBoundsException exception) {
             throw new SystemException(ExceptionType.EXCEPTION_MISSING_TAG);
         }
 
         // Split into the tag name and tag color.
-        String[] tagInfo = tagMessage[1].split(" ", 2);
+        String[] tagInfo = tagsInfo.split(" ", 2);
```

Comment:
If we are splitting by space, should we use a whitespace regex?
```suggestion
        String[] tagInfo = tagsInfo.split("\\s+", 2);
```